src/js/src/components/ToolboxComponents/LinkGraph.vue

```diff
@@ -1,24 +1,318 @@
+      // get the year - which is a nice method.
+      const year = time.getFullYear()
+      return day + '-' + month + '/' + year
+    },
+    mouseoverNode(node) {
+      d3.select(node).select('text').transition()
+                .duration(300)
+                .attr('x', 10)
+                .style('fill', '#002E70')
+                .style('stroke', 'null')
+                .style('stroke-width', '.5px')
+                .style('font', '40px sans-serif')
+              if(d3.select(node).select('circle').attr('style') !== 'fill: red;') {
+                d3.select(node).select('circle').transition()
+                  .duration(300)
+                  .attr('r', 15)
+                  .style('fill', ' #002E70')
+              }
+    },
+    highlightNode(node) {
+      d3.select(node).select('text').transition()
+                .duration(300)
+                .attr('x', 10)
+                .style('fill', '#002E70')
+                .style('stroke', 'null')
+                .style('stroke-width', '.5px')
+                .style('font', '40px sans-serif')
+              if(d3.select(node).select('circle').attr('style') !== 'fill: red;') {
+                d3.select(node).select('circle').transition()
+                  .duration(300)
+                  .attr('r', 16)
+                  .style('fill', ' #002E70')
+              }
+    },
+    normalNode(node) {
+      d3.select(node).select('text').transition()
+                .duration(300)
+                .attr('x', null)
+                .style('fill', 'black')
+                .style('stroke', 'null')
+                .style('stroke-width', '1px')
+                .style('font', '16px "Trebuchet MS", Ubuntu')
+              if(d3.select(node).select('circle').attr('style') !== 'fill: red;') {
+                d3.select(node).select('circle').transition()
+                  .duration(300)
+                  .attr('r', 5)
+                  .style('fill', 'black')
+                }
+    },
+    irrelevantNode(node) {
+      d3.select(node).select('text').transition()
+                .duration(300)
+                .attr('x', null)
+                .style('fill', '#ddd')
+                .style('stroke', 'null')
+                .style('stroke-width', '1px')
+                .style('font', '16px "Trebuchet MS", Ubuntu')
+              if(d3.select(node).select('circle').attr('style') !== 'fill: red;') {
+                d3.select(node).select('circle').transition()
+                  .duration(300)
+                  .attr('r', 5)
+                  .style('fill', '#ddd')
+                }
+    },
+    buildSvg(result) {
+      document.getElementById('graphContainer').innerHTML = ''
+      let _this = this
+      var width =  document.getElementById('graphContainer').offsetWidth, height = document.getElementById('graphContainer').offsetHeight
+      var svg = d3.select('#graphContainer').append('svg')  
```

Comment:
You changed all others to _let_ - what about these two?

Reply:
fixed now!